docs/gdb-debugging.md

```diff
@@ -108,3 +108,7 @@ command in the GDB session which will terminate both.
 
 - Currently we support a limited subset of cpu registers for get and set
   operations, if more are required feel free to contribute.
+
+- Some assumptions around virtual address configuration on arm have been made.
```

Comment:
should we add a brief description of these assumptions?

Reply:
Added a description now thanks